README.md

```diff
@@ -9,7 +9,7 @@ go get github.com/ConductorOne/conductorone-sdk-go
 <!-- End SDK Installation [installation] -->
 
 <!-- Start SDK Example Usage [usage] -->
-## SDK Example Usage
+## SDK Example Usage with Custom Server/Tenant
```

Comment:
99.9% of customer don't need a custom server/tenant, it shouldn't be the first/main example.